released/discovery_center/mission_4172/step1/main.tf

```diff
@@ -158,83 +151,84 @@ resource "btp_subaccount_subscription" "hana_cloud_tools" {
 # Assign users to Role Collection: SAP HANA Cloud Administrator
 resource "btp_subaccount_role_collection_assignment" "hana_cloud_admin" {
   for_each             = toset(var.hana_cloud_admins)
```

Comment:
proposed changes here
remove the "for_each ... " line and 
replace below line ```user_name = each.value``` with ```user_name = var.hana_system_admin```